variables.js

```diff
@@ -0,0 +1,17 @@
+function variables() {
+
+    let userinput;
+    userinput = document.getElementById("user").value;
+
+    let passinput;
+    passinput = document.getElementById("pass").value;
+
+    let GFG_Var;
+ 
+    userdisplay = (userinput);
```

Comment:
What does this accomplish?

Reply:
i think when i was displaying the user input on the webpage it helped me pull the input and then let me display it.